gfc/financial_companion/templates/pages/individual_group.html

```diff
@@ -1,80 +1,54 @@
-        </div>
-        {% if is_owner %}
-        <div class="col-12 col-md-4 d-flex justify-content-start justify-content-md-end">
-          <a href="{% url 'edit_user_group' pk=group.id %}"><button type="button" class="btn btn-success px-3 mr-2 py-1 mb-md-4">Edit Details</button></a>
-          <a href="{% url 'delete_user_group' pk=group.id %}"><button type="button" class="btn btn-danger px-3 mx-2 py-1 mb-md-4" onclick="return confirm('Are you sure you want to delete this user group?')">Delete</button></a>
-        </div>
-        {% endif %}
-      </div>
-    </div>
+{% block page_header_model %}Group{% endblock %}
+{% block page_header_name %}{{group.name.title}}{% endblock %}
 
-    <div class="row">
-      <div class="container">
-        <div class="row">
-          <div class="card">
-            <div class="card-body">
-              <p class="card-text">
-                {%if group.group_picture.url != None %}
-                  <img src="{{ group.group_picture.url }}" alt="Profile picture of {{ group.name }}" class="border border-primary rounded center " width="150" height="150">
-                {% else %}
-                  <p class="card-text"><Strong>Group Picture:</Strong> Your group picture has not been uploaded yet</p>
-                {% endif %}
-                <p><Strong>Join code: </Strong> {{group.invite_code}}</p>
-                <p><Strong>Description: </Strong> {{group.description}}</p>
-                <p><Strong>Number of members: </Strong> {{count}}</p>
-              </p>
-            </div>
+{% block card_title %}Group Information:{% endblock %}
 
-            <div class="card-footer text-muted">
-              If you have any questions regarding the group please contact the owner at: <strong>{{group.owner_email}}</strong>
-            </div>
+{% block edit_delete_buttons %}
+  {% if is_owner %}
+    {% url 'edit_user_group' pk=group.id as edit_group %}
+    {% url 'delete_user_group' pk=group.id as delete_group %}
+    {% include "partials/view_models/edit_and_delete_models_buttons.html" with edit_URL=edit_group delete_URL=delete_group model_name="group"%}
+  {% endif %}
+{% endblock %}
 
-          </div>
-        </div>
-      </div>
-    </div>
+{% block confirmation_model_name %}transaction{% endblock %}
 
-  </div>
-</div>
+{% block table_content %}
+  <li class="list-group-item"><strong>Join code: </strong> {{group.invite_code}}</li>
+  <li class="list-group-item"><strong>Description: </strong> {{group.description}}</li>
+  <li class="list-group-item"><strong>Number of members: </strong> {{count}}</li>
+{% endblock %}
 
-<br>
+{% block table_footer %}
+  If you have any questions regarding the group please contact the owner at: <strong>{{group.owner_email}}</strong>
+{% endblock %}
 
-<div style="text-align:center">
-  <a href="{% url 'individual_group' pk=group.id leaderboard=True %}"><button type="button" class="btn btn-primary px-3 mx-2 py-1 mb-md-4">View Leaderboard</button></a>
```

Comment:
These buttons are used for the users to view their leaderboard. These need to be added to the update view. @Splagon

Reply:
I see that this error has been made during partialisation and has been swiftly resolved.